pkg/utils/parse_arf_result.go

```diff
@@ -450,13 +488,79 @@ func remediationsFromString(scheme *runtime.Scheme, name string, namespace strin
 		})
 	}
 
-	return rems
+	return rems, nil
+}
+
+//This function will take orginal remediation content, and a list of all values found in the configMap
+//It will processed and substitue the value in remediation content, and return processed Remediation content
+//The return will be Processed-Remdiation Content, Value-Used List, Un-Set List, and err if possible
+func parseValues(remContent string, resultValues map[string]string) (string, []string, []string, error) {
+	t, err := template.New("").Option("missingkey=zero").Parse(remContent)
+	var valuesUsedList []string
+	var valuesMissingList []string
+	var valuesParsedList []string
+	if err != nil {
+		return remContent, valuesUsedList, valuesMissingList, errors.New("wrongly formatted remediation context: " + err.Error()) //Error creating template // Wrongly formatted remediation context
```

Comment:
Wouldn't it be nicer to use `errors.Wrap` instead of concatenating error strings?